tests/Unit/DataStructures/DataBox/Test_DataBoxTag.cpp

```diff
@@ -167,3 +167,20 @@ static_assert(
         db::split_tag<Prefix<Tags::Variables<tmpl::list<Var, Prefix<Var>>>>>,
         tmpl::list<Var, Prefix<Var>>>,
     "Failed testing split_tag");
+
+/// [prefix_tag_wraps_specified_tag]
+static_assert(db::prefix_tag_wraps_specified_tag<PrefixWithArgs<Var, int, int>,
+                                                 Var>::value,
+              "failed testing prefix_tag_wraps_specified_tag");
+
+static_assert(not db::prefix_tag_wraps_specified_tag<Prefix<Var2>, Var>::value,
```

Comment:
For the documentation, include the simpler case which evaluates to true_type and reorder.